code/datums/materials/basemats.dm

```diff
@@ -497,3 +497,18 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	beauty_modifier = 0.2
 	turf_sound_override = FOOTSTEP_WOOD
 	texture_layer_icon_state = "bamboo"
+
+/datum/material/zaukerite
+	name = "zaukerite"
+	desc = "A light absorbing crystal"
+	color = COLOR_ALMOST_BLACK
+	categories = list(MAT_CATEGORY_RIGID = TRUE, MAT_CATEGORY_ITEM_MATERIAL=TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/zaukerite
+	value_per_unit = 0.45
+	armor_modifiers = list(MELEE = 0.9, BULLET = 0.9, LASER = 1.75, ENERGY = 1.75, BOMB = 0.5, BIO = 1, RAD = 1.75, FIRE = 0.1, ACID = 1)
+	beauty_modifier = 0.001
+
+/datum/material/zaukerite/on_accidental_mat_consumption(mob/living/carbon/victim, obj/item/source_item)
+	victim.apply_damage(30, BURN, BODY_ZONE_HEAD, wound_bonus = 5)
+	source_item?.reagents?.add_reagent(/datum/reagent/toxin/plasma, source_item.reagents.total_volume*5)
```

Comment:
this intended to be plasma?

Reply:
yeah i figured that it get processed into plasma